powershell-yaml.psm1

```diff
@@ -290,17 +337,37 @@ function ConvertTo-Yaml {
             $wrt = New-Object "System.IO.StringWriter"
         }
 
+        $builder = New-Object "YamlDotNet.Serialization.SerializerBuilder"
+    
         if ($PSCmdlet.ParameterSetName -eq 'NoOptions') {
-            $Options = 0
-
             if ($JsonCompatible) {
                 # No indent options :~(
-                $options = [YamlDotNet.Serialization.SerializationOptions]::JsonCompatible
```

Comment:
The enum still exists in YamlDotNet, so you can leave this here, along with ```$Options = 0```. See bellow for continuation.